behave/features/e2e_test_features/09.s3_outputs_check.feature

```diff
@@ -0,0 +1,8 @@
+# COVID-19 -
+@s3_outputs_check
+Feature: COVID-19 Shielded vulnerable people service - validate data from previous day
+    Scenario: Pipeline should be executed and data validated for previous day run
+        Given the data is available for yesterday
+        When the pipeline is processed
```

Comment:
I think this should be:

```
Given there was a new web submission yesterday
and the pipeline has completed successfully
When the output CSVs are checked
Then ...
```


```

Reply:
I'll update the Given to confirm a web submission yesterday.

When clause though I think should be file processed. With proposed structure, we will need to check file in when clause implementation as well which will be redundant as we need to check files in then clause for test nhs user.